.github/workflows/build_package.yml

```diff
@@ -9,7 +9,7 @@ jobs:
         strategy:
             matrix:
                 os: [macos-latest, ubuntu-latest, windows-latest]
-                python: [3.8, 3.9]
+                python: [3.9]
```

Comment:
also let it run with python 3.10 now

Reply:
I put 3.10 in there before, but somehow github tests for 3.1. Since it doesnt't work and the build fails I took it out for now.

Reply:
there you go: https://github.com/actions/runner/issues/1989 seems like you need to add single quotes around it.